src/main/java/seedu/address/logic/commands/NoteCommand.java

```diff
@@ -68,8 +53,7 @@ public CommandResult execute(Model model) throws CommandException {
      * {@code personToEdit}.
      */
     private String generateSuccessMessage(Person personToEdit) {
-        String message = !note.value.isEmpty() ? MESSAGE_ADD_NOTE_SUCCESS : MESSAGE_DELETE_NOTE_SUCCESS;
-        return String.format(message, personToEdit);
+        return String.format(MESSAGE_ADD_NOTE_SUCCESS, personToEdit);
```

Comment:
Message returned doesn't display NoteDate... Also NoteDate is a good variable name, but I wonder if it will be intuitive for the user?
> `Opened note to Person: Bernice Yu; Phone: 99272758; Email: berniceyu@example.com; NoteDate: ; Tags: [colleagues][friends]`

Actually, on a UI note I wonder @HalphasX if it would be nice if the last edited notes date showed up **next to** the name of the individual